code/modules/reagents/chemistry/reagents.dm

```diff
@@ -54,6 +54,8 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	var/overrides_metab = 0
 	/// above this overdoses happen
 	var/overdose_threshold = 0
+	///Overrides what addiction this chemicals feeds into, allowing you to have multiple chems that treat a single addiction.
+	var/datum/reagent/addiction_type
```

Comment:
why does this need to be an instanced item?

Reply:
actually it doesnt has to be